commit 1ab5258f7c2abfafcd63a760ca08bf93591912da
Author: Khem Raj <raj.khem@gmail.com>
Date:   Wed Dec 14 16:11:05 2016 -0800

    Avoid conflicts with integer width macros from TS 18661-1:2014
    
    glibc 2.25+ has now defined these macros in <limits.h>
    https://sourceware.org/git/?p=glibc.git;a=commit;h=5b17fd0da62bf923cb61d1bb7b08cf2e1f1f9c1a
    
    Create an alias for FC_CHAR_WIDTH for ABI compatibility
    
    Signed-off-by: Khem Raj <raj.khem@gmail.com>

diff --git a/fontconfig/fontconfig.h b/fontconfig/fontconfig.h
index 5c72b22..070a557 100644
--- a/fontconfig/fontconfig.h
+++ b/fontconfig/fontconfig.h
@@ -128,7 +128,8 @@ typedef int		FcBool;
 #define FC_USER_CACHE_FILE	    ".fonts.cache-" FC_CACHE_VERSION
 
 /* Adjust outline rasterizer */
-#define FC_CHAR_WIDTH	    "charwidth"	/* Int */
+#define FC_CHARWIDTH	    "charwidth"	/* Int */
+#define FC_CHAR_WIDTH	    FC_CHARWIDTH
 #define FC_CHAR_HEIGHT	    "charheight"/* Int */
 #define FC_MATRIX	    "matrix"    /* FcMatrix */
 
diff --git a/src/fcobjs.h b/src/fcobjs.h
index 1fc4f65..d27864b 100644
--- a/src/fcobjs.h
+++ b/src/fcobjs.h
@@ -51,7 +51,7 @@ FC_OBJECT (DPI,			FcTypeDouble,	NULL)
 FC_OBJECT (RGBA,		FcTypeInteger,	NULL)
 FC_OBJECT (SCALE,		FcTypeDouble,	NULL)
 FC_OBJECT (MINSPACE,		FcTypeBool,	NULL)
-FC_OBJECT (CHAR_WIDTH,		FcTypeInteger,	NULL)
+FC_OBJECT (CHARWIDTH,		FcTypeInteger,	NULL)
 FC_OBJECT (CHAR_HEIGHT,		FcTypeInteger,	NULL)
 FC_OBJECT (MATRIX,		FcTypeMatrix,	NULL)
 FC_OBJECT (CHARSET,		FcTypeCharSet,	FcCompareCharSet)
